integrations/GoogleOptimize/browser.js

```diff
@@ -0,0 +1,49 @@
+import logger from "../../utils/logUtil";
+import ScriptLoader from "../ScriptLoader";
+
+class GoogleOptimize {
+  constructor(config) {
+    this.name = "GOOGLE_OPTIMIZE";
+    this.trackingId = config.trackingId;
+    this.containerId = config.containerId;
+    this.async = config.async;
+  }
+
+  init() {
+    logger.debug("===in init Google Optimize===");
+    if (!this.trackingId) {
+      return;
+    }
+    ScriptLoader(
+      "Google Tag Manager",
+      `https://www.googletagmanager.com/gtag/js?id=${this.trackingId}`
+    );
+    window.dataLayer = window.dataLayer || [];
+    function gtag() {
+      dataLayer.push(arguments);
+    }
+    gtag("js", new Date());
+    gtag("config", `${this.trackingId}`);
+
+    if (!this.containerId) {
+      return;
+    }
+    if (this.async) {
+      ScriptLoader(
+        "Google Optimize",
+        `https://www.googleoptimize.com/optimize.js?id=${this.containerId}`
+      );
```

Comment:
We also need to add support for anti-flicker code (to be configured from the control plane)

Reply:
Anti flicker things are not absolutely necessary since it is required for very small audience like those who use Google Ads etc. Can we check with our customer if they need that? , else it will be unnecessary load on website.

Reply:
Let's add this as a setting in the config as discussed